src/gtk/control.cpp

```diff
@@ -49,6 +49,10 @@ bool wxControl::Create( wxWindow *parent,
 {
     bool ret = wxWindow::Create(parent, id, pos, size, style, name);
 
+#ifdef __WXGTK3__
+    m_cachedPreferredSize = wxSize(wxDefaultCoord, wxDefaultCoord);
```

Comment:
I'm not sure if we want to return this (known as `wxDefaultSize` BTW) or the default value (i.e. `wxSize(0,0)`) from `GetBestSize()` if we really don't have anything. A priori, the latter seems slightly better to me, why did you decide to use `wxDefaultSize` instead?

In any case, this merits a comment (unless the code is completely removed, of course, but even in this case we may want to add a comment about this to the variable declaration in the header).